apps/trading/components/sidebar/sidebar.tsx

```diff
@@ -104,6 +111,14 @@ export const Sidebar = () => {
         </Routes>
       </nav>
       <nav className={classNames(navClasses, 'ml-auto lg:mt-auto lg:ml-0')}>
+        <SidebarButton
```

Comment:
Do we need to disable this button, or display different content in the dialog when already connected?

Reply:
Added "disabled" state to the SidebarButton